gcp/image.pkr.hcl

```
@@ -13,7 +22,7 @@ variable "zone" {
 source "googlecompute" "hashistack" {
   image_name   = "hashistack-${local.timestamp}"
   project_id   = var.project
-  source_image = "ubuntu-minimal-1804-bionic-v20221026"
+  source_image = "ubuntu-minimal-2204-jammy-v20231213b"
```

```suggestion
  source_image = "ubuntu-minimal-2204-jammy-v20240318"
```
Newer base image available